specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/virtualMachineScaleSet.json

```diff
@@ -3489,6 +3489,79 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/virtualmachines/{instanceId}/attachDetachDataDisks": {
+      "post": {
+        "tags": [
+          "VirtualMachineScaleSetVMs"
+        ],
+        "operationId": "VirtualMachineScaleSetVMs_AttachDetachDataDisks",
+        "description": "Attach and detach data disks to/from a virtual machine in a VM scale set.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
```

Comment:
use the latest "v5" common type:

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/parameter-not-using-common-types.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/parameter-not-using-common-types.md)

Reply:
@ms-zhenhua Currently in Microsoft.Compute I cannot see the version v5 specified anywhere . I can see even in main we are in v1 version. Can you please elaborate how v5 can be specified .